walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -14,10 +14,85 @@
 
 package com.google.sps;
 
+import java.util.Arrays;
+import java.util.ArrayList;
+import java.util.Comparator;
 import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
 
 public final class FindMeetingQuery {
+  /**
```

Comment:
can you add some more comments about the edge cases? like if there are no mandatory attendees but are optional attendees, what happens? should also mention that optional attendees are first considered when finding available times, but they are removed from consideration if no available times are found with them considered

javadocs are useful for people that might come along later, and have no idea what this method is supposed to do. they can read the javadoc and know what the method promises to do without having to read the source code. stating what happens during edge cases is extremely helpful